tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>Please use <code><strong>getDownloadURL</strong></code><strong> </strong> (cap URL) to be consistent with how the API docs capitalize.</li><li>Use only <code>lyon</code>  for the 'region' argument.</li><li>Remove the output cell.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Done</p>